### ISSUE TITLE
Healthcheck broke setups with basic auth enabled

### ISSUE THREAD

**Issue by michaelarnauts** [Tautulli/Tautulli-Docker#21]
#18 broke my container. I have authentication set to http basic auth, and it returns a 401 to the curl call, therefore stating that my container is not healthy, and therefore my proxy is refusing to connect.

```
$ curl -ILfSs http://localhost:8181 > /dev/null || curl -ILfkSs https://localhost:8181 > /dev/null || echo "this is wrong"
curl: (22) The requested URL returned error: 401 Unauthorized
curl: (35) error:1408F10B:SSL routines:ssl3_get_record:wrong version number
this is wrong
root@8365337f71bb:/$
```

**Comment by michaelarnauts** [comment]
I've added the following to the service in my docker-compose.yml file in the meantime:
```yaml
    healthcheck:
      disable: true
```

**Comment by TheMeanCanEHdian** [comment]
We are adding a new [status check](https://github.com/Tautulli/Tautulli/commit/4743538de88f51fb836deac7f15f1f056de10a0e) that bypasses auth that should resolve this issue. The container won't be updated until the change has moved to stable and people have had time to update.

**Comment by TheMeanCanEHdian** [comment]
This is fixed in e9305acbcae479787c61cd69df68d079bea4941e
